docs/reference/primitives/rack/README.md

```diff
@@ -0,0 +1,63 @@
+# Rack
+
+A Rack is a platform to build, deploy and host your [Apps](../app).  It runs either locally on your own machine for development purposes or within your cloud infrastructure.
+
+## Command Line Interface
+
+### Installing a Rack
```

Comment:
Let's drop the install instructions here, they are already going to propagate several places and will likely rot here